djarg/tests/test_integration.py

```diff
@@ -146,6 +146,36 @@ def test_grant_staff_object_view(client):
     assert not user.is_staff
 
 
+@pytest.mark.django_db
+def test_grant_staff_if_same_name_object_view(client):
```

Comment:
This is pretty much a copy/paste of the above test - tho I removed some "basic" tests that the other test took care of.